### ISSUE TITLE
10 K ohm vs 9.53K ohm

### ISSUE THREAD

**Issue by BurtHarris** [alwynallan/redoubler#4]
The schematic diagram in the readme shows the values of R4, R6, R8, and R10 as 9.53K, but the block diagram shows these as all 10K.   Is this critical?

**Comment by alwynallan** [comment]
My early prototypes had all 10K resistors and no voltage regulator. I found
frequent glitches in the data, depending on the quality of the power. Bill
Cox had the same experience with his similar circuit, and solved it by
reducing gain using 8.2K in the feedback path. He had a 3.3V regulator all
along. The cost is that the random bits become more correlated (less
entropy per bit) as the gain is reduced below 2.0.

I found that 9.53K with a regulator avoided problems in all cases that I
tested. A lower resistor value would avoid problems more reliably.

Let me know if I missed something, or new questions come up.

A. Peter Allan · 450 Timber Lane · Devon PA 19333 · 610-687-1708 ·
610-547-4836 mobile
